src/components/CldPlaceholder/CldPlaceholder.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <img v-if="!isImageLoaded()" class="cld-placeholder" :src="src" :width="this.width" :height="this.height"/>
+</template>
+<script>
+  import {getPlaceholderURL} from "../../helpers/getPlaceholderURL";
+
+  export default {
```

Comment:
If I have 

```
<cld-image>
   <cld-placeholder type='blur'/>
   <cld-transformation "angle"=30/>
</cld-image>
```

Will the placeholder image be rotated?

Reply:
It should because we take all parent transformations and just append the 'placeholder' mod at the end.